Sources/MapboxNavigation/UIImage.swift

```diff
@@ -43,4 +43,29 @@ extension UIImage {
 
         return scaledImage
     }
+
+    func tint(_ tintColor: UIColor) -> UIImage {
```

Comment:
Can't we just use UIImageRenderingModeAlwaysTemplate? I should do something similar.